platform/ui/src/components/FilterDateRange/FilterDateRange.jsx

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import { DateRange, FilterWrapper } from '@ohif/ui';
+
+const FilterDateRange = ({
+  label,
+  isSortable,
+  isBeingSorted,
+  sortDirection,
```

Comment:
Should this sort stuff be composable? Part of some HOC magic?

Reply:
I think we can cut this down to:

`isSortable` and `sortDirection`
It's being sorted if `isSortable` is true and `sortDirection` is `ascending` or `descending` (or whatever their corresponding values are)

Reply:
Updated :)